source/nodejs/adaptivecards-designer/src/adaptivecards-designer.css

```diff
@@ -242,6 +242,26 @@
     font-weight: bold;
 }
 
+.acd-open-code-dialog-container {
+    display: flex;
+}
+
+.acd-open-code-dialog-preview-container {
+    flex: 1;
+}
+
+.acd-open-code-dialog-editor-container {
+    flex: 1;
+}
+
+.acd-open-code-dialog-preview {
```

Comment:
This class is empty...do we need it or can we delete this style definition?

Reply:
Resolved: deleted this class

Reply:
While empty classes should in general be removed, the JS SDK (renderer and designer) tends to leave empty classes in the CSS so as to make it clear they exist. Essentially, until we have better documentation, the CSS is the reference when it comes to the list of available classes.